### PR TITLE
Feature wls life cycle

### PR DESCRIPTION
**Pull request by vikashsonu900**

### CODE REVIEW

File: ansible-fmw/fmw_wls_lifecycle.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- name: Play to start the services of WLS domain
+  hosts: dojoserver
+  become: true
+  become_user: "{{ WLS_USER }}"
+  vars_files:
+    - '{{ playbook_dir }}/wls_variables.yml'
+  tasks:
+    - name: Lifecycle start
+      tags: start
+      block:
```
Comment (zeusbaba) [inline ansible-fmw/fmw_wls_lifecycle.yml:11]: fyi. `nm` should be started before `adminserver` 
and for stop the order is reverse

Comment (zeusbaba) [inline ansible-fmw/fmw_wls_lifecycle.yml]: fyi. `nm` should be started before `adminserver` 
and for stop the order is reverse

File: ansible-fmw/roles/dojo-wls-lifecycle/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+Role Name
```
Comment (zeusbaba) [inline ansible-fmw/roles/dojo-wls-lifecycle/README.md]: plz kindly update `README` , this content is autogenerated ;)

File: ansible-fmw/wls_variables.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@ JDK_DIR: '{{ JAVA_DIR }}/jdk1.8.0_331'
 JAVA_HOME: '{{ JAVA_DIR }}/java_latest'
 
 ## Oracle ENV Variables
-ORACLE_BASE: "/home/dojowls/wls_lite"
+ORACLE_BASE: "/home/dojowls/day-3"
```
Comment (zeusbaba) [inline ansible-fmw/wls_variables.yml:18]: IMHO we should avoid using custom names like `day-3`  
the should be generic and reusable, e.g. `wls-server` 
remember that idempotency is important aspect of IaC

File: ansible-fmw/roles/dojo-wls-lifecycle/README.md (hunk 2 above)
Comment (zeusbaba) [inline ansible-fmw/roles/dojo-wls-lifecycle/README.md]: plz kindly update README , this content is autogenerated ;)

Comment (zeusbaba) [inline ansible-fmw/roles/dojo-wls-lifecycle/README.md]: plz kindly update README , this content is autogenerated ;)

File: ansible-fmw/fmw_wls_lifecycle.yml (hunk 1 above)
Comment (zeusbaba) [inline ansible-fmw/fmw_wls_lifecycle.yml]: fyi. `nm` should be started before `adminserver`
and for stop the order is reverse

Comment (zeusbaba) [inline ansible-fmw/fmw_wls_lifecycle.yml]: fyi. `nm` should be started before `adminserver`
and for stop the order is reverse

**Review by zeusbaba — CHANGES_REQUESTED** [review]
very good job!  
plz kindly see feedback

File: ansible-fmw/fmw_wls_lifecycle.yml (hunk 1 above)
Comment (vikashsonu900) [reply ansible-fmw/fmw_wls_lifecycle.yml]: This has been incorporated.

File: ansible-fmw/roles/dojo-wls-lifecycle/README.md (hunk 2 above)
Comment (vikashsonu900) [reply ansible-fmw/roles/dojo-wls-lifecycle/README.md]: This was a blank role, so we have deleted it and pushed again.

File: ansible-fmw/wls_variables.yml (hunk 3 above)
Comment (vikashsonu900) [reply ansible-fmw/wls_variables.yml:18]: This has been changed to `ORACLE_BASE: "/home/{{ WLS_USER }}/fmw" `

File: ansible-fmw/roles/dojo-wls-lifecycle/README.md (hunk 2 above)
Comment (vikashsonu900) [reply ansible-fmw/roles/dojo-wls-lifecycle/README.md]: We have deleted the blank role.

**Pull request closed without merging**